tests/aws/scenario/test_loan_broker.py

```diff
@@ -0,0 +1,259 @@
+"""
+This scenario test is taken from https://github.com/localstack-samples/sample-loan-broker-stepfunctions-lambda
+which in turn is based on https://www.enterpriseintegrationpatterns.com/ramblings/loanbroker_stepfunctions.html
+"""
+import json
+import os
+from dataclasses import dataclass
+
+import aws_cdk
+import aws_cdk as cdk
+import aws_cdk.aws_dynamodb as dynamodb
+import aws_cdk.aws_lambda as awslambda
+import aws_cdk.aws_logs as logs
+import aws_cdk.aws_stepfunctions as sfn
+import aws_cdk.aws_stepfunctions_tasks as tasks
+import pytest
+
+from localstack.testing.aws.util import is_aws_cloud
+from localstack.testing.pytest import markers
+from localstack.testing.scenario.provisioning import InfraProvisioner
+from localstack.utils.strings import short_uid
+from localstack.utils.sync import retry
+
+RECIPIENT_LIST_STACK_NAME = "LoanBroker-RecipientList"
+PROJECT_NAME = "CDK Loan Broker"
+OUTPUT_LOAN_BROKER_STATE_MACHINE_ARN = "LoanBrokerArn"
+OUTPUT_LOAN_BROKER_LOG_GROUP_NAME = "LogGroupName"
+LOAN_BROKER_TABLE = "LoanBrokerBanksTable"
+
+CREDIT_BUREAU_JS = "./resources_loan_broker/bank_app_credit_bureau.js"
+BANK_APP_JS = "./resources_loan_broker/bank_app.js"
+
+
+def _read_file_as_string(filename: str):
+    file_path = os.path.join(os.path.dirname(__file__), filename)
+
+    content = None
+    with open(file_path, "r") as file:
+        content = file.read()
+    return content
+
+
+@dataclass
+class Bank:
+    bank_id: str
+    base_rate: str
+    max_loan: str
+    min_credit_score: str
+
+    def get_env(self) -> str:
```

Comment:
the return value here doesn't seem correct :thinking:

Reply:
oopsi 🤦‍♀️